docs/user_guide/storing/index_hnswlib.md

```diff
@@ -208,4 +208,41 @@ To delete nested data, you need to specify the `id`.
 del doc_index[index_docs[6].id]
 ```
 
-Check [here](docindex#nested-data-with-subindex) for nested data with subindex.
\ No newline at end of file
+Check [here](docindex#nested-data-with-subindex) for nested data with subindex.
+
+### Update elements
+In order to update a Document inside the index, you only need to reindex it with the updated attributes.
+
+```python
+import numpy as np
+
+from docarray import BaseDoc, DocList
+from docarray.typing import NdArray
+from docarray.index import HnswDocumentIndex
+
+class MyDoc(BaseDoc):
+    text: str
+    embedding: NdArray[128]
+
+docs = DocList[MyDoc](
```

Comment:
I'd show an example where we retrieve some data (find, filter..) and replace only those docs, otherwise I don't think this is a relatable example